.github/workflows/ci.yml

```diff
@@ -806,6 +814,19 @@ jobs:
         echo "#include <rz_util.h>`nint main(int argc, char **argv) { return rz_str_newf (`"%s`", argv[0]) != NULL? 0: 1; }" > test.c
         cl -IC:$env:HOMEPATH\AppData\Local\Programs\rizin\include\librz -IC:$env:HOMEPATH\AppData\Local\Programs\rizin\include\librz\sdb /Fetest.exe test.c /link /libpath:C:$env:HOMEPATH\AppData\Local\Programs\rizin\lib rz_util.lib
         .\test.exe
+    - name: Check CMake config files
+      shell: pwsh
+      run: |
+        .\dist\windows\vsdevenv.ps1 ${{ matrix.bits }}
+        $env:PATH = "$env:PATH;C:$env:HOMEPATH\AppData\Local\Programs\rizin\bin"
+        $env:PKG_CONFIG_PATH = "C:$env:HOMEPATH\AppData\Local\Programs\rizin\lib\pkgconfig"
+        $env:PKG_CONFIG_PATH
+        echo -e "#include <rz_util.h>\nint main(int argc, char **argv) { return rz_str_newf (\"%s\", argv[0]) != NULL? 0: 1; }" > test.c
+        echo -e "cmake_minimum_required(VERSION 3.10)\nproject(RizinCMakeTest VERSION 1.0.0)\nfind_package(RzUtil)\nadd_executable(rizin_cmake_test test.c)\ntarget_link_libraries(rizin_cmake_test PRIVATE ${RzUtil_TARGET})\n" > CMakeLists.txt
```

Comment:
Same

Reply:
```
Run .\dist\windows\vsdevenv.ps1 64
C:\Users\runneradmin\AppData\Local\Programs\rizin\lib\pkgconfig
Write-Output: D:\a\_temp\28bc5b82-ee80-48e5-8c98-7b64bae4b4a7.ps1:6
Line |
   6 |  echo -e "#include <rz_util.h>\nint main(int argc, char **argv) { retu …
     |       ~~
     | Parameter cannot be processed because the parameter name 'e' is ambiguous. Possible matches include:
     | -ErrorAction -ErrorVariable.
```